src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -222,6 +212,29 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontWeight: 700,
     marginTop: "1rem",
   },
+  itemContainer: {
+    height: '100vh',
```

Comment:
might need to reduce the height as the current gap between the header and the carousel is a bit too much

<img width="1761" alt="image" src="https://github.com/Switcheo/carbon-website/assets/88943108/131ba903-7f26-4f7c-a765-8ecb3c3b2afc">

Reply:
Replaced the use of VH values, so this should be the same on all screens now.